src/IStringDictionary.ts

```diff
@@ -83,4 +84,66 @@ export interface StringDictionary {
   teamSubmittedAdminNotification: stringFunction;
   teamSubmittedpostMessage: stringFunction;
   /*----------------------------------*/
+
+  /*----------------------------------*/
+  // src>discord>events>message>exit.ts
+  exitFlow: string;
+  /*----------------------------------*/
+
+  /*----------------------------------*/
+  // src>discord>events>message>help.ts
+  prizeMessage: string;
+  appInfoMessage: string;
+  interactMessage: string;
+  /*----------------------------------*/
+
+  /*----------------------------------*/
+  // src>discord>events>message>index.ts
+  botTaggedMsg: string;
+  botCantUnderstand: string;
+  genericErrorMessage: string;
+  helpDescript: string;
+  pingDescript: string;
+  ideaDescript: string;
+  techDescript: string;
+  jobDescript: string;
+  registerDescript: string;
+  exitDescript: string;
+  /*----------------------------------*/
+
+  /*----------------------------------*/
+  // src>discord>events>message>registerTeam.ts
+  registerNotOpen: string;
+  registerTeamUserMessage: string;
+  registerTeamTitle: string;
+  registerTeamJudingTitle: string;
+  registerTeamDescription: string;
+  registerTeamExit: string;
+  registerTeamExitValue: string;
+  AppNameMessage: string;
+  AppDetailMessage: string;
+  AppChannelMessage: string;
+  teamChannel: string;
+  finalTeamTitle: string;
+  finalTeamDescription: string;
+  TeamNameTitle: string;
+  TeamMembersTitle: string;
+  TeamDescriptionTitle: string;
+  TeamChannelName: string;
+  DuplicateChannelCode: string;
+  DuplicateChannel: string;
+  FailedSavingTeamLogger: string;
+  FailedSavingTeam: string;
```

Comment:
Could you change these to start with a lower case letter to match the format of the rest?